Sources/Sentry/SentryBinaryImageCache.m

```diff
@@ -99,6 +99,19 @@ - (NSInteger)indexOfImage:(uint64_t)address
     return -1; // Address not found
 }
 
+- (nullable NSString *)pathForImage:(NSString *)binaryName
+{
+    @synchronized(self) {
+        for (SentryBinaryImageInfo *info in _cache) {
+            if ([[info.name.lastPathComponent lowercaseString]
+                    isEqualToString:binaryName.lowercaseString]) {
+                return info.name;
+            }
+        }
+    }
```

Comment:
`m`: since this uses a cache, I'm assuming this is not performance-heavy?

Reply:
I wouldn't consider this performance-heavy, but this is just used once, so definitely not a problem.

Reply:
FYI @kahest: The cache is for keeping a record of the binary images. It gets updated when you add and remove binary images.